gazebo.repos

```diff
@@ -0,0 +1,65 @@
+repositories:
```

Comment:
this file is a copy of this one https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-garden.yaml

Reply:
This is the right one https://raw.githubusercontent.com/gazebo-tooling/gazebodistro/master/collection-garden.yaml